test/IntegrationTests/Helpers/TimeExtensions.cs

```diff
@@ -15,20 +15,23 @@
 // </copyright>
 
 using System;
-using IntegrationTests.Helpers.Constants;
 
 namespace IntegrationTests.Helpers;
 
 internal static class TimeExtensions
 {
+    private const long NanoSecondsPerTick = 1000000 / TimeSpan.TicksPerMillisecond;
```

Comment:
nit: this is just moved from a deleted file but given that it is a constant it seems better to just say `100` instead of the division.

Reply:
TBH for me the current code is more readable. Otherwise I would not know from where this knowledge comes from. The compiler converts it to `100` anyway.

Reply:
perhaps I have stamped in my brain that it is 100ns ;) - fine for me.